### ISSUE TITLE
[BUG] Problem with nested list item's indentation

### ISSUE THREAD

**Issue by atipugin** [st3v3nmw/obsidian-spaced-repetition#800]
**Describe the bug**
Indentation level gets lost after review in case if card is nested inside a list. E.g.:

```markdown
- foo
    - bar?::baz
```

transforms into: 

```markdown
- foo
- bar?::baz <!--SR:!2023-11-25,1,230-->
```

**To Reproduce**

1. Create a card nested inside a list.
2. Review the card.

**Expected behavior**

Plugin shouldn't change item's indentaton and only append comment with metadata.

```markdown
- foo
    - bar?::baz <!--SR:!2023-11-25,1,230-->
```

**Screenshots**

None.

**Versions (please complete the following information):**

-   OS: macOS 14.0
-   Obsidian version: 1.4.16
-   Plugin version: 1.10.5
-   If on desktop, Installer version: 1.4.13

**Additional context**

None.

**Comment by jajaperson** [comment]
Getting this on iOS two, only began recently though.

**Comment by ronzulu** [comment]
Sorry, that could have been my doing when making the code tolerant of leading/trailing spaces.

I'll have a look, hopefully today.

Cheers
Ronny

**Comment by ronzulu** [comment]
Hi  @atipugin &  @jajaperson 

Sorry about the delay, here is an update ready for beta testing.

There is a file called `main.js` in this zip file
[main_800A.zip](https://github.com/st3v3nmw/obsidian-spaced-repetition/files/13631081/main_800A.zip)

You will need to copy that to the obsidian plug-in folder, e.g.
 `D:\Obsidian\Obsidian\.obsidian\plugins\obsidian-spaced-repetition`

Then reload the plug-in. Please let me know how it goes.

Ronny

**Comment by ronzulu** [comment]
Hi @atipugin & @jajaperson

Did you get a chance to check the version from my previous post? If ok, I'll put in the merge request.

**Comment by atipugin** [comment]
Hey @ronzul,

Looks like the issue is gone!
<img width="413" alt="image" src="https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/281812/b0361cfd-aca7-4917-8f9d-64c777a1839e">

**Comment by ronzulu** [comment]
Hi @atipugin the fix has been included in the latest release 1.11.0. Please close this at your convenience.

Cheers
Ronny

**Comment by atipugin** [comment]
@ronzulu great!
